test/distributed/pipeline/sync/test_pipe.py

```diff
@@ -622,7 +637,7 @@ def forward(self, x):
 
             return x
 
-    model = nn.Sequential(DelayedLog(0, seconds=0), DelayedLog(1, seconds=0.1))
+    model = nn.Sequential(DelayedLog(0, seconds=0), DelayedLog(1, seconds=0.1).to(0))
```

Comment:
Not sure I followed why we're adding `.to(0)` in many test files.

Reply:
Son in these cases there are multiple consecutive partitions all on cpu. I need to move some layers to gpu, otherwise partitions are all merged into one single partition and test does not work as intended.